config/vufind/config.ini

```diff
@@ -840,6 +847,14 @@ authors         = Wikipedia
 url = "https://api.vlb.de/api/v1/cover/"
 ; token = "XXXXXX-XXXX-XXXXX-XXXXXXXXXXXX"
 
+; This section is needed for LocalFile cover loading.
+; Note:  All coverimage LocalFile:ConfigSection must include the following keys: 
+;        filepath, filename, and imagetype.
+[LocalFile]
+filepath = /usr/local/vufind/local/coverimages
+filename = recordid
```

Comment:
It might also be helpful to make filename an optionally multi-valued setting so you could try multiple things, like:

<pre>
filename[] = "%size%/isbn/%isbn%"
filename[] = "%size%/issn/%issn%"
</pre>